src/pages/documentation/adobe-analytics/api-reference.md

```diff
@@ -133,6 +133,24 @@ iOS
 
 <Tabs query="platform=ios&api=get-visitor-identifier-with-completion-handler"/>
 
+## registerExtension
+
+Registers the Analytics extension with the Mobile Core SDK.
+
+<InlineAlert variant="warning" slots="text"/>
+
+Deprecated as of 2.0.0. Use the [MobileCore.registerExtensions](../mobile-core/api-reference.md#registerextensions) API instead.
```

Comment:
Oh yeah, one more very minor thing - can we make it "Please use the..." instead? (I like being 🇨🇦 and polite.)

Reply:
Sure!